### ISSUE TITLE
pp not defined

### ISSUE THREAD

**Issue by patlx** [Taapeli/isotammi-addons#28]
Hi,

I've installed V5.2-2 two days ago and I get the following error when trying to start SuperTool:

```python
Traceback (most recent call last):
  File "C:\Program Files\GrampsAIO64-5.2.2\gramps\gen\plug\_manager.py", line 274, in load_plugin
    _module = self.import_plugin(pdata)
              ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\GrampsAIO64-5.2.2\gramps\gen\plug\_manager.py", line 308, in import_plugin
    module = __import__(pdata.mod_name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\patri\AppData\Local\gramps\gramps52\plugins\SuperTool\SuperTool.py", line 103, in <module>
    pp(a)
    ^^
NameError: name 'pp' is not defined
2024-05-10 11:40:39.353: WARNING: utils.py: line 231: Failed to open addon metadata for fr https://raw.githubusercontent.com/prculley/isotammi-addons/master/addons/gramps52/listings/addons-fr.json: HTTP Error 404: Not Found
Traceback (most recent call last):
  File "C:\Program Files\GrampsAIO64-5.2.2\gramps\gen\plug\_manager.py", line 274, in load_plugin
    _module = self.import_plugin(pdata)
              ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\GrampsAIO64-5.2.2\gramps\gen\plug\_manager.py", line 338, in import_plugin
    module = __import__(pdata.mod_name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\patri\AppData\Local\gramps\gramps52\plugins\SuperTool\SuperTool.py", line 103, in <module>
    pp(a)
    ^^
NameError: name 'pp' is not defined
```

**Comment by kkujansuu** [comment]
Please try to install the latest version of Supertool (1.4.2). This was a bug in an earlier version but it should be fixed now.

**Comment by emyoulation** [comment]
This was reported again in Oct.2024

fixed in the SuperTool 1.3.3 version.
see https://github.com/Taapeli/isotammi-addons/issues/18
